samples/AzureFunctionsApp/Entities/counters.http

```diff
@@ -0,0 +1,26 @@
+@host = http://localhost:7071/api
```

Comment:
I assume these files require a VS Code extension to run? Should we put a comment in here making that clear, and perhaps pointing them to the download location?

Reply:
added vscode recommended extensions.